content/2022-05-19-leave-the-yaml-at-home.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Leave the YAML at Home!"
+slug: "2022-05-19-leave-the-yaml-at-home"
+description: "Did you know Brigade involves a minimal amount of YAML?"
+date: "2022-05-19"
+author: "Kent Rancourt"
+summary: "We hear people aren't always crazy about YAML. Have you met Brigade?"
+---
+
+Hello again, Brigadiers!
+
+We've heard from time to time that people aren't always so crazy about YAML --
+and that's putting it nicely. I don't personally harbor such strong anti-YAML
+sentiment, but the critics are out there and as critics are wont to be, they're
+quite vocal. If you dare to search for "yaml sucks" in your favorite search
+engine, you'll find a litany of criticism -- some of it valid. So we thought
```

Comment:
I think you should briefly describe why some of the criticism is valid. Otherwise, I don't think the `some of it valid` part adds much.

Reply:
I think I was just trying to neither invalidate nor buy into the (general) criticism of YAML. I don't hate YAML, but I'm sure those who do make some solid points.

I think this comment has actually given me a lot to think about... If I don't actually hate YAML so much, why do I see our minimal YAML as a selling point? I guess my _real_ problem is that I don't see a declarative language as expressive as an imperative one for modeling workflows.

I think I need to sell that idea a little harder without latching on to more generalized anti-YAML sentiment.